packages/app/src/providers/hsds/hsds-api.ts

```diff
@@ -43,6 +43,7 @@ import {
   convertHsdsAttributes,
   convertHsdsShape,
   convertHsdsType,
+  flattenValue,
```

Comment:
`flattenValue` and `sliceValue` are no longer needed by the `@h5web/h5wasm` package, so I've moved them closer to where they are still used (in `hsds-api` and `mock-api`).